python/cudf/cudf/testing/dataset_generator.py

```diff
@@ -48,16 +48,19 @@ def __init__(
         self,
         cardinality=100,
         null_frequency=0.1,
-        generator=lambda: [
-            _generate_string(string.ascii_letters, random.randint(4, 8))
-            for _ in range(100)
-        ],
+        generator=None,
         is_sorted=True,
         dtype=None,
     ):
         self.cardinality = cardinality
         self.null_frequency = null_frequency
-        self.generator = generator
+        if generator is None:
+            self.generator = lambda: [
+                np.random.default_rng(seed=0).integers(0, 100)
+                for _ in range(100)
+            ]
+        else:
+            self.generator = generator
```

Comment:
This seems to be changing the default generator?

Reply:
Fixed this.